src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -712,6 +712,11 @@ private uint getMethodAttribsInternal(MethodDesc method)
                 }
             }
 
+            if (method.HasCustomAttribute("System.Runtime.CompilerServices", "IntrinsicAttribute"))
```

Comment:
Dtto

Reply:
Hmm, don't we need a check similar to what Crossgen1 has at

https://github.com/dotnet/runtime/blob/5bee16dae8bdc44731542566a5343aba900bbad2/src/coreclr/src/vm/methodtablebuilder.cpp#L5139

?

Reply:
That's what `MethodDesc.IsIntrinsic` property does in crossgen2.

Reply:
OK, thanks for clarification, deleted in 4th commit. What about the CORINFO_FLG_JIT_INTRINSICS flag, should I also put it back? It seemed superfluous to me but I may be overlooking something.

Reply:
Yes, you should put it back. Otherwise, the new-style intrinsics won't work.

Reply:
OK, put back in 5th commit, thanks for clarification.